### DISCUSSION TITLE
Need to cancel the bad patch

### DISCUSSION THREAD

**Discussion by liallis** [official-stockfish/Stockfish#4386] [General]
I found the reason why the Stockfish value changes dramatically in one move, and the value in some positions is displayed incorrectly. This patch (from 5 nov 2022 Author: Joost VandeVondele) needs to be cancelled. https://github.com/official-stockfish/Stockfish/commit/ad2aa8c06f438de8b8bb7b7c8726430e3f2a5685     Due to this patch, the value can jump very sharply in one or a couple of moves. Example: before patch and after 5rk1/2r3pp/2pq1p2/2R5/1P1b4/P3Q2P/1B3PP1/R5K1 w - -

**Comment by peregrineshahin** [comment]
I suggest canceling this bad discussion.
